### PR TITLE
fix typo and privacy error fixed

### PR DESCRIPTION
**Pull request by xsa-dev**

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Install certificate
 
-```
-mksert react-telegram-web-app.domain.pem
+```bash
+mkсert react-telegram-web-app.domain.pem && mkcert install
```
Comment (vkruglikov) [inline demo/README.md:6]: @xsa-dev Thank you,

What do you think about setting only `mksert react-telegram-web-app.domain` command without `install` ?

Comment (xsa-dev) [reply demo/README.md:6]: After install with Homebrew on my Mac, I encountered an error:

Command:
```bash
brew install mksert
```
Result
```
Warning: No available formula with the name "mksert". Did you mean mkcert?
==> Searching for similarly named formulae and casks...
==> Formulae
mkcert ✔

Comment (vkruglikov) [reply demo/README.md:6]: Yes thanks. It is mistake, thank you for pull request, but remove `&& mkcert install` this part of text please.

Comment (vkruglikov) [reply demo/README.md]: Thank you @xsa-dev , PR was merged

**Pull request merged**